### PR TITLE
Many changes to get webpack for server and client

### PR DESCRIPTION
**Pull request by mmahalwy**
### Title of change
Not a rewrite! Many of the files have been borrowed from https://github.com/ctrlplusb/react-universally. 

### Checklist

- [ ] Unit tests written
- [ ] Manually tested
- [ ] Prettier & ESLint were run
- [ ] New dependencies are included in `package-lock.json`

### Screenshot
_insert screenshot if needed_

### CODE REVIEW

**Comment by mmahalwy** [comment]
Ugh, mostly moving files but it's freaking out about line changes...

File: src/client/styles/components/MasterHeader.scss [hunk 1]
```diff
@@ -0,0 +1,86 @@
+$content-padding: 10 15px 5px 15px !important;
```
Comment (mmahalwy) [inline src/client/styles/components/MasterHeader.scss]: This is jsut moved

**Comment by ahmedre** [comment]
rebuild

**Comment by ahmedre** [comment]
rebuild

**Comment by ahmedre** [comment]
rebuild

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32768

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32769

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32770

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32771

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32772

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32773

**Comment by mmahalwy** [comment]
Going to merge this so I can iterate quickly for Ramadan

**Pull request merged**